docs/reference/api/volume_ls_types.md

```diff
@@ -0,0 +1,8 @@
+# VolumeLsOptions
+
+| Field     | CLI flag        | Description                                                                             |
+| --------- | --------------- | --------------------------------------------------------------------------------------- |
+| `Quiet`   | `--quiet`,`-q`  | Only display volume names                                                               |
+| `Format`  | `--format`      | Format the output using the given go template                                           |
+| `Size`    | `--size`,`-s`   | Display the disk usage of volumes. Can be slow with volumes having loads of directories |
+| `Filters` | `--filter`,`-f` | Filter matches volumes based on given conditions                                        |
```

Comment:
Isn't godoc enough?

cc @Zheaoli

Reply:
I'm unsure if we need to describe the struct field.  If we need to describe the struct field, I think godoc is not enough.